maro/requirements.build.txt

```diff
@@ -1,2 +1,5 @@
+numpy==1.19.1
 Cython==0.29.14
-numpy==1.19.1
\ No newline at end of file
+altair==4.1.0
+streamlit==0.69.1
```

Comment:
why change this file?

Reply:
Changing the location of numpy is to pass the check of editorconfig.
 Arthur advised me to do so.